Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml

```diff
@@ -111,17 +108,28 @@ script:
     - defaultValue: '0'
       description: Page in which to retrieve results.
       name: page
-    description: Searches for security rules using the SIQL language query (limit to 10k).
+    description: Searches for security rules using the SIQL language query (limit to 10k)
     name: firemon-secmgr-secrule-search
     outputs:
     - contextPath: FireMonSecurityManager.SIQL.matchId
       description: Resposne for the SIQL query.
       type: Unknown
-  dockerimage: demisto/python3:3.10.13.80014
+  - arguments:
+    - name: page
+    - name: pageSize
+    description: Get all the collectors in the inventory.
+    name: firemon-collector-get-all
```

Comment:
```suggestion
    name: firemon-list-collectors
```

Reply:
I would leave the Naming Convention as is since the Firemon's API-DOC (swagger ui) organizes the RestAPI call's by
1) Modules, i.e. Collectors, Domain, License and etc.
2) Within each module, the API calls are referred by https method, i.e. Get, Put and etc.
So if anyone who is using Firemon's API-DOC, the name conversion of "firemon-module-httpmethod" would be considered as a natural fit.